staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/testing/testcase.go

```diff
@@ -517,10 +517,14 @@ func NewMutatingTestCases(url *url.URL) []Test {
 				NamespaceSelector:       &metav1.LabelSelector{},
 				AdmissionReviewVersions: []string{"v1beta1"},
 			}},
-			ExpectAllow:       true,
-			AdditionalLabels:  map[string]string{"remove": "me"},
-			ExpectLabels:      map[string]string{"pod.name": "my-pod"},
-			ExpectAnnotations: map[string]string{"removelabel.example.com/key1": "value1"},
+			ExpectAllow:      true,
+			AdditionalLabels: map[string]string{"remove": "me"},
+			ExpectLabels:     map[string]string{"pod.name": "my-pod"},
+			ExpectAnnotations: map[string]string{
+				"removelabel.example.com/key1":                              "value1",
+				"mutation.webhook.admission.k8s.io/removelabel.example.com": "true",
+				"patch.webhook.admission.k8s.io/removelabel.example.com":    `[{"op": "remove", "path": "/metadata/labels/remove"}]`,
```

Comment:
you had some reinvocation code. How will it look like for the second (or third) invocation?

Reply:
the key will be suffix with index to distinguish the round
```
mutation.webhook.admission.k8s.io/removelabel.example.com-1: true
mutation.webhook.admission.k8s.io/removelabel.example.com-2: false
patch.webhook.admission.k8s.io/removelabel.example.com-1: ...
```